src/_sass/post.scss

```diff
@@ -27,8 +34,32 @@
         border-color: $near-white-lh;
     }
 
+    a {
+        color: $pink-lh;
+    }
+
+    p {
+        margin-bottom: 0.5em;
+    }
+
     blockquote {
         color: $light-gray-lh;
-        padding-bottom: 2em;
+        padding-bottom: 1.5em;
     }
-}
\ No newline at end of file
+}
+
+.post--video-container {
+        position: relative;
+        overflow: hidden;
+        padding-top: 56.25%;
```

Comment:
Where did you get this number from? 😛

Reply:
Hehe, I just searched "responsive iframe" and found that: https://blog.theodo.com/2018/01/responsive-iframes-css-trick/

It looked very good!